static/app/views/performance/landing/widgets/components/widgetContainer.tsx

```diff
@@ -174,16 +135,19 @@ const _WidgetContainer = (props: Props) => {
 export const WidgetContainerActions = ({
   setChartSetting,
   allowedCharts,
+  rowChartSettings,
 }: {
-  loading: boolean;
   setChartSetting: (setting: PerformanceWidgetSetting) => void;
   allowedCharts: PerformanceWidgetSetting[];
+  rowChartSettings: PerformanceWidgetSetting[];
 }) => {
   const organization = useOrganization();
   const menuOptions: React.ReactNode[] = [];
 
+  const inactiveCharts = allowedCharts.filter(chart => !rowChartSettings.includes(chart));
```

Comment:
We typically prefer to disable the options from the dropdown rather than hide them.

Reply:
Right, good call.